public/style.css

```diff
@@ -15,3 +15,15 @@
 ul {
   list-style: none;
 }
+
+.green {
+  color: green;
+}
+
+.red {
+  color: red;
+}
+
+.default {
```

Comment:
Are we using default class anywhere at the moment?

Reply:
No, removed in the last commit